pinot-query-runtime/src/main/java/org/apache/pinot/query/service/dispatch/DispatchClient.java

```diff
@@ -58,4 +90,26 @@ public void cancel(long requestId) {
     Worker.CancelRequest cancelRequest = Worker.CancelRequest.newBuilder().setRequestId(requestId).build();
     _dispatchStub.cancel(cancelRequest, NO_OP_CANCEL_STREAM_OBSERVER);
   }
+  private SslContext buildSslContext(TlsConfig tlsConfig) {
```

Comment:
lack of new line and duplicate code. You can perhaps create a new utils class in `org.apache.pinot.common.utils.grpc`?

cc: @Jackie-Jiang if he has a better idea.

Reply:
@ankitsultana can we take it as a follow-up ?
As It is used in multiple places - It will cause touching other files too.

Reply:
Hmm that should be fine. I think we should avoid duplicating code in this PR itself.

Reply:
Okay - let me update the PR then

Reply:
@ankitsultana @Jackie-Jiang 
Let me know if this looks good , or if you have any different thought for the utility.